StableMotifs/Format.py

```diff
@@ -230,13 +262,23 @@ def import_primes(fname, format='BooleanNet', remove_constants=False):
 
     """
     # TODO: add more formats
-    rules = remove_comment_lines(open(fname))
-    if format == 'BooleanNet':
-        rules = booleannet2bnet(rules)
-    elif format == 'BNet':
-        rules = rules
+    if format == 'CellCollective':
+        rules1 = remove_comment_lines(open(fname + '/expr/expressions.ALL.txt'))
+        external = remove_comment_lines(open(fname + '/expr/external_components.ALL.txt'))
+        lines = external.splitlines()
+        for i in range(len(lines)):
+            lines[i] = lines[i] + ' = ' + lines[i]
+        rules2 = "\n".join(lines)
+        rules = rules1 + "\n" + rules2
+
```

Comment:
I think you need to add `rules = cellcollective2bnet(rules)` here, because `longbnet2primes` expects the bnet format, but it looks like rules is still in the cell collective format.